test/jdk/java/awt/GraphicsConfiguration/NonDefaultGC.java

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright (c) 2002, 2024, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/*
+ * @test
+ * @bug         4131642
+ * @summary     This test shows the ability to create Frames, Windows
+ * and Canvases with a GraphicsConfiguration. The test should show a number
+ * of windows with RGB stripes in according to the number of the
+ * GraphicsConfigurations for each screen. It also displays the size of
+ * the screen and the GraphicsConfiguration.toString().
```

Comment:
Better to align the sentences after @summary tag words so that @tags can be on one column and other sentences do not encroach, similar to test/jdk/sun/java2d/pipe/DrawImageBgTest.java in this PR